content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -39,6 +39,11 @@ memory or CPU resources must be requested as well.
 A pod may consume multiple huge page sizes in a single pod spec. In this case it
 must use `medium: HugePages-<hugepagesize>` notation for all volume mounts.
 
+Note that this Volume and VolumeMount are only necessary if your application
+requests huge pages via the `mmap` syscall, without the `MAP_HUGETLB` flag. If your
```

Comment:
Following the [style guide(https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases), avoid Latin abbreviations.
```suggestion
requests huge pages with the `mmap` syscall and without the `MAP_HUGETLB` flag. If your
```